app/models/event.rb

```diff
@@ -1,6 +1,6 @@
 class Event < ActiveRecord::Base
   include ActiveRecord::Transitions
-  has_paper_trail
+  has_paper_trail ignore: [:updated_at, :guid, :week], meta: { conference_id: :conference_id }
```

Comment:
Should that **not** show me week in changes? (I am seeing it on updating an event)